### PR TITLE
Charles/school

### PR DESCRIPTION
**Pull request by charlesx2013**
Add radius to school

### CODE REVIEW

**Comment by kelseylam** [comment]
Bad sentence

File: app/controllers/api/v1/admin/schools_controller.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def destroy
   private
 
   def school_params
-    params.require(:school).permit(:name, :lat, :lng, :address)
+    params.require(:school).permit(:name, :lat, :lng, :address, :radius)
```
Comment (sameerav) [inline app/controllers/api/v1/admin/schools_controller.rb]: requirements are a social construct
stop being so oppressive

File: app/models/school.rb [hunk 2]
```diff
@@ -9,6 +9,7 @@
 #  lat        :decimal(10, 7)   default(0.0)
 #  lng        :decimal(10, 7)   default(0.0)
 #  address    :string
+#  radius     :integer          default(200)
```
Comment (sameerav) [inline app/models/school.rb]: WHO DEFINES DEFAULTS REALLY
everyone is special and unique

File: app/serializers/schools/base_school_serializer.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class BaseSchoolSerializer < ActiveModel::Serializer
-  attributes :id, :name, :lat, :lng, :address, :student_count
+  attributes :id, :name, :lat, :lng, :address, :radius, :student_count
```
Comment (sameerav) [inline app/serializers/schools/base_school_serializer.rb]: is this the radius of how much your breastmilk spreads

Comment (sameerav) [inline app/serializers/schools/base_school_serializer.rb]: please let it be that radius

File: app/serializers/users/base_user_serializer.rb [hunk 4]
```diff
@@ -21,6 +21,8 @@ def volunteer_type
   end
 
   def user_semester
+    return unless object
```
Comment (sameerav) [inline app/serializers/users/base_user_serializer.rb]: stop objectifying people
jesus christ

File: app/serializers/users/user_list_serializer.rb [hunk 5]
```diff
@@ -2,6 +2,6 @@ class UserListSerializer < BaseUserSerializer
   has_one :school, serializer: SchoolNameSerializer
 
   def school
-    object.school
+    object.school if object
```
Comment (sameerav) [inline app/serializers/users/user_list_serializer.rb]: AGAIN WITH THE OBJECTIFICATION

**Pull request closed without merging**